### ISSUE TITLE
Fb links does not work properly

### ISSUE THREAD

**Issue by boxdxnx** [inputconf/Input-site#75]
Links such as `fb.me/exampletext` at section `#team` does not work properly

`https://inputconf.org/fb.me/oleksiy.rudenko.5` 
```
404 
File not found
The site configured at this address does not contain the requested file.
If this is your site, make sure that the filename case matches the URL.
For root URLs (like http://example.com/) you must provide an index.html file.
Read the full documentation for more information about using GitHub Pages.
```

**Comment by lenadgit** [comment]
link of PR  
https://github.com/inputconf/Input-site/pull/76/commits/3ad73ff66a09d591a191692667226398dcf2a04e

**Comment by A-Ostrovnyy** [comment]
Closed by [PR#76](https://github.com/inputconf/Input-site/pull/76)
